### PR TITLE
Add label_error on object detection

### PR DESCRIPTION
**Pull request by crsegerie**

### CODE REVIEW

File: utils/debug.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 if __name__ == "__main__":
     # The command you want to debug
-    cmd = "train --project-id XXX"
+    cmd = "label_errors --project-id cl5tqv0xkcv6b0nvr3qng5tqx  --asset-status-in LABELED"
```
Comment (crsegerie) [inline utils/debug.py:12]: rm

File: commands/label_errors.py [hunk 2]
```diff
@@ -106,14 +220,13 @@ def main(
...
+    jobs = curated_job(jobs, target_job, ignore_job)
 
     for job_name, job in jobs.items():
-        if target_job and job_name not in target_job:
-            continue
         kili_print(f"Detecting errors for job: {job_name}")
         content_input = get_content_input_from_job(job)
-        ml_task: MLTaskT = job.get("mlTask")  # type: ignore
-        tools: List[ToolT] = job.get("tools")
+        ml_task = job.get("mlTask")
```
Comment (PierreLeveau) [inline commands/label_errors.py:228]: is there a reason why the types are removed?

File: kiliautoml/utils/helpers.py [hunk 3]
```diff
@@ -319,3 +320,20 @@ def is_contours_detection(input_type, ml_task, content_input, tools):
         and ml_task == "OBJECT_DETECTION"
         and any(tool in tools for tool in ["semantic", "polygon"])
     )
+
+
+def curated_job(jobs: JobsT, target_job: List[JobNameT], ignore_job: List[JobNameT]) -> JobsT:
```
Comment (PierreLeveau) [inline kiliautoml/utils/helpers.py:325]: can you add a docstring ? hard to guess what it does.

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: commands/label_errors.py (hunk 2 above)
Comment (crsegerie) [reply commands/label_errors.py:228]: No longer needed

Comment (crsegerie) [reply commands/label_errors.py:228]: job is now better typed

File: commands/label_errors.py [hunk 4]
```diff
@@ -26,37 +32,141 @@
...
     ModelRepositoryT,
     ProjectIdT,
-    ToolT,
 )
 
 
-def upload_errors_to_kili(error_recap: ErrorRecap, kili):
-    kili_print("Updating metadatas for the concerned assets")
+def upload_errors_to_kili(error_recap: ErrorRecap, kili: Kili, project_id: ProjectIdT):
+    kili_print("\nUpdating metadatas for the concerned assets")
```
[the remark this answers is not in the document]
Comment (crsegerie) [reply commands/label_errors.py:42]: metadatum, metadati, metadatibus... errare humanum est

File: requirements.txt [hunk 5]
```diff
@@ -27,3 +26,5 @@ opencv-python
 git+https://github.com/facebookresearch/detectron2.git
 shapely
 pydantic
+pygit2
```
[the remark this answers is not in the document]
Comment (crsegerie) [reply requirements.txt:29]: Yes you are right, no longer needed since I've uninstalled tensorflow

**Review by daniellopezhe — APPROVED** [review]

**Pull request merged**